src/query/storages/fuse/src/operations/read/parquet_data_source_deserializer.rs

```diff
@@ -62,6 +72,46 @@ pub struct DeserializeDataTransform {
     virtual_reader: Arc<Option<VirtualColumnReader>>,
 
     base_block_ids: Option<Scalar>,
+    cached_runtime_filter: Option<Expr>,
+}
+
+#[macro_export]
+macro_rules! impl_runtime_filter {
```

Comment:
Why use the macro here?

Reply:
`NativeDeserializeDataTransform` and `DeserializeDataTransform` will share the method